src/components/Layout.tsx

```diff
@@ -1,16 +1,31 @@
 import * as React from "react";
-import PropTypes from "prop-types";
-
 import { Header } from "@graydigital/storybook";
-import logo from "../images/gray_digital_logo_white.png";
-
+import { useStaticQuery, graphql } from "gatsby";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
 
 export const Layout: React.FC = ({ children }) => {
+  const data = useStaticQuery(graphql`
```

Comment:
Cool this hook is new since last time I've used Gatsby.

Reply:
yes, def comes in handy for functional components and when used outside pages